src/CollAction/ValidationAttributes/MinImageDimensionsAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿using CollAction.Helpers;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+
+namespace CollAction.ValidationAttributes
+{
+    public class MinImageDimensionsAttribute : ValidationAttribute
+    {
+        private readonly int _minWidth;
+        private readonly int _minHeight;
+
+        public MinImageDimensionsAttribute(string widthByHeight)
```

Comment:
Why not two int parameters? Seems a bit akward taking a string, which is not type-safe, and then parsing it to two dimensions.